sunpy/image/coalignment.py

```diff
@@ -306,6 +325,7 @@ def repair_image_nonfinite(image):
         the next non-finite value is replaced by the mean of its finite
         valued nearest neighbors.
     """
+    warnings.warn('This function has been deprecated.', SunpyDeprecationWarning)
```

Comment:
You should instead use the `deprecated` decorator in `sunpy.util.decorators` to mark this function as deprecated.

Reply:
done!